cmd/slurm.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const slurmTemplate string = `#!/bin/bash
+#SBATCH --job-name={{.JobName | shquote}}
+#SBATCH --output=slurm.out
+#SBATCH --export=ALL
+{{- if .Config.Parallel}}
+#SBATCH --ntasks={{.Config.Threads}}{{end}}
+#SBATCH --chdir={{.WorkingDirectory | shquote}}
```

Comment:
Outside the context of Metworx, I suspect callers may end up needing to set additional parameters.  I'm not sure exactly how that should look (and am not proposing it for this PR), but, as mentioned in 4e7df052 (sge: convert qsub options to template directives, 2024-12-06), I think this series gets us closer to making that happen:

> And now that submission happens with a plain 'qsub {script}' call, it
> opens up the possibility of allowing callers to override the template
> (e.g., via a command-line argument).